src/mofaflex/_core/mofaflex.py

```diff
@@ -133,6 +136,15 @@ class ModelOptions(_Options):
     nonnegative_weights: dict[str, bool] | bool = False
     """Non-negativity constraints for weights for each view (if dict) or for all views (if bool)."""
 
+    guiding_vars_weight_priors: dict[str, str] | str | None = None
+    """Weight prior for each guiding variable (if dict) or for all guiding variables (if str)."""
+
+    guiding_vars_likelihoods: dict[str, str] | str | None = None
+    """Likelihood for each guiding variable (if dict) or for all guiding variables (if str)."""
+
+    guiding_vars_likelihood_scales: dict[str, float] | float | None = None
+    """Likelihood scale for each guiding variable (if dict) or for all guiding variables (if float)."""
```

Comment:
Do we need that? We scale the standard view likelihoods to account for views with different numbers of features, but all guiding variables are 1-dimensional.